packages/appstore/src/assets/svgs/trading-platform/ic-appstore-swapfree.svg

```diff
@@ -0,0 +1 @@
72 4.608v19.922h-6.217z" fill="#0A559E"/><path d="M8.609 11.973c.168.31.36.694.576 1.154.225.45.455.942.69 1.476.243.525.482 1.064.717 1.617l.66 1.561.661-1.56c.235-.554.47-1.093.704-1.618.243-.534.473-1.026.689-1.476.225-.46.421-.844.59-1.154h1.997c.094.647.178 1.374.253 2.18.085.797.155 1.631.211 2.503.066.863.122 1.73.169 2.602.056.872.103 1.692.14 2.46H14.53c-.028-.946-.066-1.977-.112-3.093a83.532 83.532 0 00-.211-3.375l-.563 1.308-.619 1.434a128.15 128.15 0 01-1.054 2.49h-1.533c-.131-.31-.29-.68-.478-1.112l-.59-1.378a93.55 93.55 0 00-.606-1.434l-.562-1.308a83.586 83.586 0 00-.211 3.375 253.183 253.183 0 00-.113 3.094H5.74c.037-.769.08-1.59.127-2.461l.168-2.602c.066-.872.136-1.706.211-2.503.085-.806.174-1.533.267-2.18H8.61zM25.03 11.973v1.87h-2.94v7.876h-2.193v-7.875h-2.94v-1.87h8.072zM29.674 18.85c0-.272-.061-.51-.183-.717-.122-.216-.333-.394-.633-.535-.3-.14-.703-.248-1.21-.323-.506-.075-1.143-.112-1.912-.112.103-.872.188-1.758.253-2.658.066-.9.117-1.744.155-2.532h5.274v1.758h-3.516c-.019.338-.042.666-.07.985-.02.309-.043.58-.07.815 1.368.094 2.376.417 3.023.97.656.544.984 1.304.984 2.279 0 .45-.08.867-.239 1.251-.16.385-.399.717-.717.999-.319.28-.722.501-1.21.66-.478.16-1.04.24-1.687.24-.253 0-.52-.02-.802-.057a7.52 7.52 0 01-.801-.126 8.579 8.579 0 01-.69-.155 2.706 2.706 0 01-.477-.169l.38-1.73c.234.104.543.207.927.31.385.094.844.14 1.378.14.647 0 1.116-.126 1.407-.379.29-.262.436-.567.436-.914zM13.253 34.458c.155 0 .281-.011.38-.035a.807.807 0 00.246-.105c.061-.047.103-.101.127-.162a.563.563 0 00.035-.204c0-.16-.075-.29-.225-.394-.15-.108-.408-.222-.774-.344a6.735 6.735 0 01-.478-.19 1.917 1.917 0 01-.429-.274 1.37 1.37 0 01-.309-.401c-.08-.16-.12-.352-.12-.577 0-.225.043-.426.127-.604a1.29 1.29 0 01.359-.464c.154-.127.342-.223.562-.289.22-.07.469-.105.745-.105.329 0 .612.035.851.105.24.07.436.148.59.232l-.316.865a2.727 2.727 0 00-.457-.183 1.854 1.854 0 00-.597-.084c-.263 0-.453.037-.57.113-.112.07-.169.18-.169.33 0 .089.021.164.064.225.042.06.1.117.175.169.08.047.17.091.268.133.103.038.215.078.337.12.253.094.474.187.661.281.188.09.342.195.464.316.127.122.22.265.281.43.061.163.092.363.092.597 0 .455-.16.809-.478 1.062-.319.248-.8.372-1.442.372-.215 0-.41-.014-.583-.042a2.91 2.91 0 01-.809-.21 2.85 2.85 0 01-.239-.12l.31-.872c.145.08.323.152.534.218.215.06.478.091.787.091zM19.218 30.943c.056.192.117.41.183.653l.217.746.24.773c.08.258.157.497.232.718.06-.24.121-.5.182-.781.066-.281.127-.572.183-.872.056-.3.11-.598.162-.893.056-.3.108-.588.155-.865h1.139a47.978 47.978 0 01-.57 2.482c-.21.84-.44 1.636-.689 2.391h-1.005c-.164-.45-.324-.921-.479-1.413-.154-.497-.307-.994-.457-1.49l-.45 1.49c-.15.492-.307.963-.47 1.413h-1.006a37.46 37.46 0 01-.696-2.39 53.096 53.096 0 01-.563-2.483h1.189a51.09 51.09 0 00.33 1.758c.06.3.122.59.183.872.065.281.131.541.197.78.084-.23.166-.473.246-.73.08-.259.157-.516.232-.774.075-.258.143-.507.204-.746l.175-.64h.936zM22.616 35.295v-4.873h3.262v.921h-2.166v1.02h1.927v.921h-1.927v2.011h-1.096z" fill="#fff"/></svg>
```

Comment:
maybe use `viewBox="0 0 72 72"` instead of `width="38" height="39"` becuase it's causing the icon to not scale and fit properly: ![image](https://user-images.githubusercontent.com/108507236/227093992-6f3daccd-9c88-4f07-8911-ddf48509b0a9.png)

Reply:
Its generated from the SVG Go and trying the viewboc instead of width and height is making it more smaller.